### DISCUSSION TITLE
issue: Field Validation not workign Properly

### DISCUSSION THREAD

**Discussion by rajagopalstalwart** [react-hook-form/react-hook-form#11700] [Q&A]
### Version Number

7.51.1

### Codesandbox/Expo snack

https://nphk3z.csb.app

### Steps to reproduce

1. select the month and year for my field and submit the data, but its showing error message.

### Expected behaviour

im expected as if user selects the date ranges then submit the data in console.log inside

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by sujinleeme** [comment]
The link is not accessible. Could you check?
